mmengine/optim/scheduler/param_scheduler.py

```diff
@@ -1409,11 +1409,21 @@ def __init__(self,
             raise ValueError('Factor should be < 1.0.')
         self.factor = factor
 
+        if isinstance(optimizer, OptimWrapper):
```

Comment:
Please add some comments to describe why we need to distinguish the type of optimizer here.

Reply:
This part of the code is coupled with the `OptimWrapper`, so it requires some comments to improve readability.

Reply:
yep, I have mentioned this operation in the additional comment of the new commit.